test/OpenTelemetry.Contrib.Instrumentation.GrpcCore.Tests/OpenTelemetry.Contrib.Instrumentation.GrpcCore.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="1.3.0" />
+    <PackageReference Include="Grpc.Tools" Version="1.17.0" />
+    <PackageReference Include="Grpc" Version="[2.32.0,3.0)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.1" />

```

Comment:
all other projects are using xunit. not a rule, but would be good to keep consistency.